test-runner/src/test.ts

```diff
@@ -50,6 +51,56 @@ export class Test {
     return this.titlePath().join(' ');
   }
 
+  slow(): void;
+  slow(condition: boolean): void;
+  slow(description: string): void;
+  slow(condition: boolean, description: string): void;
+  slow(arg?: boolean | string, description?: string) {
+    const condition = typeof arg === 'boolean' ? arg : true;
```

Comment:
I'd rather accept any truthy value that is not a string, wdyt?
`const condition = typeof arg === 'string' ? true : !!arg`;

Reply:
can do